source/rmarkdown/annelida_data_analyse.Rmd

```diff
@@ -192,41 +206,72 @@ tidy_physeq$samples %>%
 ## OTU tabel
 
 ```{r verken-otu-data}
-glimpse(otu_table(physeq) %>% as.data.frame %>% as_tibble())
+
+
+glimpse(otu_table(physeq_rarefied) %>% as.data.frame %>% as_tibble())
 ```
 
 ## Taxonomie tabel
 
+
 ```{r verken-taxonomie-data}
-glimpse(tax_table(physeq) %>% as.data.frame %>% as_tibble())
+glimpse(tax_table(physeq_rarefied) %>% as.data.frame %>% as_tibble())
+```
+
+```{r GBIF-check-presence}
+
+species <- as.vector(tax_table(physeq)[, "species"])
+species <- species[!grepl("_otu", species)]
+species <- gsub("_", " ", species)
+source(here::here("source/r/check_presence.R"))
+gbif_check <- check_presence(species)
+gbif_check %>%
+filter(!present) %>%
+  kable(caption = "Not present according to GBIF in Western-Europe")
 ```
 
 Unieke en gedeelde taxa per diepte:
 
 ```{r}
-tidy_physeq %>%
+tidy_physeq_rarefied %>%
   tidytacos::tacoplot_venn(Diepte)
 ```
 
 Unieke en gedeelde taxa per landgebruik:
 
 ```{r}
-tidy_physeq %>%
+tidy_physeq_rarefied %>%
   tidytacos::tacoplot_venn(Landgebruik_MBAG)
 ```
 
 ```{r}
-tidy_physeq %>%
+tidy_physeq_rarefied %>%
   tidytacos::tacoplot_stack()
 ```
 
 ```{r}
-tidy_physeq %>%
+# Create a new combined column
+combined_column <- paste(sample_data(physeq_rarefied)$Landgebruik_MBAG, sample_data(physeq_rarefied)$Diepte, sep = "_")
+
+# Add the new combined column to the sample metadata with the desired name
+sample_data(physeq_rarefied)$Landgebruik_MBAG_diepte <- combined_column
+
+# Update the phyloseq object with the modified sample data
+physeq_rarefied <- merge_phyloseq(physeq_rarefied, sample_data(physeq_rarefied))
+
+
+psadd::plot_krona(physeq_rarefied, "MBAG_Olig01_Annelida_rar_species_alle_stalen_Landgebruik_MBAG_per_diepte", "Landgebruik_MBAG_diepte", trim = T)
```

Comment:
Hiervoor is Kronatools installatie vereist - waarvoor je ook nog een Perl interpreter geïnstalleerd moet hebben. Mij is het niet gelukt om dit te installeren / aan de praat te krijgen. Zijn er alternatieve R packages die iets gelijkaardig doen zonder al de installatie-troubles? Indien niet, hoe heb je het aan de praat gekregen?

Reply:
ah juist, ik herinner mij iets, dat was idd geen makkelijke installatie, geen probleem op linux maar dat gaf destijds bij mij ook problemen op windows, ik hoop dat ik gedocumenteerd heb hoe ik het heb opgelost, ik kijk en laat het weten

Reply:
Dag Hans, ik vind het niet terug hoe ik dit installatie probleem heb opgelost destijds op Windows, ik zou dit er dus voorlopig uit laten. De Krona charts zijn handig omdat die interactief zijn, maar we maken ze dan wel apart

Reply:
ok heb het toegevoegd